include/mgos_crontab.h

```diff
@@ -196,6 +196,17 @@ void mgos_crontab_register_handler(struct mg_str action, mgos_crontab_cb cb,
  */
 time_t mgos_crontab_get_next_invocation(mgos_crontab_job_id_t id, time_t date);
 
+/*
+ * Deactivate all existing cron jobs
+ */
+void mgos_crontab_remove_all();
+
+/*
+ * Reads crontab from the given file, and calls ct_apply() for it.
+ * If NULL is given as json_path, default crontab.json path will be used.
+ */
+void mgos_crontab_reload_from_json(const char *json_path);
```

Comment:
since we expose remove_all, i think we can make this function append jobs from the secified file.
this way, user can get a reload by calling remove_all upfront, or can combine several crontab files if they wish.
so let's do that and call it mgos_crontab_load_from_json

also, please update the comment - users need not know what ct_apply means, so describe what the function does in general terms.